README.md

```diff
@@ -1,34 +1,52 @@
-# Gunicorn Operator
+# Gunicorn-k8s Operator
 
-## Description
+A Juju charm deploying docker images using gunicorn as default. 
 
-A charm that allows you to deploy your gunicorn application in kubernetes.
+This charm simplifies docker image deployment management, allowing us to inject variables to the environment as we see fit. It allows for deployment on
```

Comment:
Whilst we don't have a standard written for this, do you think it is worth limiting the line length to, e.g., 80 or 100 characters?